examples/src/main/scala/io/chrisdavenport/shellfish/contacts/cli/Cli.scala

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) 2024 Typelevel
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package io.chrisdavenport.shellfish.contacts.cli
+
+import cats.effect.IO
+import cats.syntax.all.*
+
+import io.chrisdavenport.shellfish.contacts.core.ContactManager
+import io.chrisdavenport.shellfish.contacts.domain.flag.*
+import io.chrisdavenport.shellfish.contacts.domain.contact.*
+
+class Cli private (cm: ContactManager) {
+
+  def addCommand: IO[Unit] =
```

Comment:
I would also modify all the methods to accept a `(cm: ContactManager)` as a first argument in its own set of parenthesis. `Cli` being a class is quite useless.

```suggestion
object Cli {

  def addCommand(cm: ContactManager): IO[Unit] =
```

This modification tuples with [this one](https://github.com/davenverse/shellfish/pull/86/files#r1730008381)